docs/FAQ.md

```diff
@@ -0,0 +1,72 @@
+# FAQ
+
+## What's the difference between `>` and `$`?
+
+`>` or `connect-terminal` runs its argument
+(which by default is your shell)
+inside a terminal,
+while `$` or `connect-shell` simply runs its argument.
+
+Some programs must be run inside a terminal,
+while others spawn their own graphical window.
+Use `>` for the first type and `$` for the second.
+
+## What's the purpose of `&` or `:connect-detach`?
+
+`connect-detach` detaches (closes) the current client
+(leaving you in a terminal if you launched kakoune there) and
+creates a file named `connect.sh` which you can run with `sh connect.sh`
+to start a shell connected to that client's server.
+The file is automatically erased when it's sourced.
+
+Note 1: The kakoune server is deleted if you detach the last client and
+you didn't start it in daemon mode.
+
+Note 2: The implementation uses kakoune's `quit!` command instead of `quit`.
+
+## What is `kak-shell`?
+
+`kak-shell` is inspired by [nix-shell].
+
+It lets you run a program (by default your shell) connected to a session.
+Initially, it displays the list of active sessions and asks you to choose one.
+
+```
+$ kak-shell
+Kakoune sessions:
+1 kanto
+2 johto
++ create new session
+Kakoune session: 1█
+@kanto $ :a█
+```
+
+If you enter a number i, it will try to connect to the i-th entry.
+Otherwise, it will interpret the entry as a session name and connect to it.
+If the session is not running, it will start it in daemon mode.
+
+Apart from running a connected shell, you can use `kak-shell` to run
+commands in the context of the session.
+For example, to print the current working directory of the server, you can run
```

Comment:
Is `pwd` useful?  `kak-shell :attach` looks more useful.

Reply:
I prefer to put that inside a recipe, because it involves more things (it spawns a session as daemon mode if it doesn't exist, it's not the same as kak -c) and it's a nice tip.

`kak-shell tig`?